README.md

```diff
@@ -45,7 +45,7 @@ Optional Authentication: Create a Personal Access Token (PAT) for authentication
     - Note: `Read-only token for OSSF Scorecard Action - myorg/myrepo` (Note: replace `myorg/myrepo` with the names of your organization and repository so you can keep track of your tokens.)
     - Expiration: `No expiration`
     - Scopes: 
-        * `repo > public_repo`                  Required to read [Branch-Protection](https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection) settings.
+        * `repo > public_repo`                  Required to read [Branch-Protection](https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection) settings. **Warning**: for private repositories, you need scope `repo`.
```

Comment:
Nit: s/Warning/Note/ to make the sentence less threatening :)

Reply:
Done, thanks